lib/KListWithOverflow.vue

```diff
@@ -195,11 +207,17 @@
           maxWidth -= removedDividerWidth;
```

Comment:
We need to also check the logic of the fix dividers Visibility, because for example here, when we have dividers, we can have this situation where some overflowed items are visible:

https://github.com/user-attachments/assets/ac572531-7806-4462-a841-45a08f04bc37